tests/php/Security/Validation/EntropyPasswordValidatorTest.php

```diff
@@ -6,6 +6,7 @@
 use SilverStripe\Dev\SapphireTest;
 use SilverStripe\Security\Member;
 use SilverStripe\Security\Validation\EntropyPasswordValidator;
+use Symfony\Component\Validator\Constraints\PasswordStrength;
```

Comment:
This isn't needed but it's not worth going through a whole review cycle for it - it's not doing any harm.
One day we will add a linting rule for unnecessary `use` statements :sweat_smile: